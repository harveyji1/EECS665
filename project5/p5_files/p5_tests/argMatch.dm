fn: (a: int, b: int) void{
    hello: int;
}

fn2: (a: bool) void{
    bye: int;
}

main: () void{
    fn(true, false);
    fn2(1);
}