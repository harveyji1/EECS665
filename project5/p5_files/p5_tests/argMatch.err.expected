FATAL [10,8]-[10,12]: Type of actual does not match type of formal
FATAL [10,14]-[10,19]: Type of actual does not match type of formal
FATAL [11,9]-[11,10]: Type of actual does not match type of formal
Type Analysis Failed
